Dockerfile.ci

```diff
@@ -0,0 +1,32 @@
+ARG image_tag=latest
+FROM pinterb/jq:0.0.16 AS jq
```

Comment:
Seems recent enough. We can mirror it like https://hub.docker.com/r/elifesciences/goaws/?